### ISSUE TITLE
Left side of focus ring around `KButton` in Device permissions is cut off

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9177]
## Observed behavior
Left side of focus ring around `KButton` in Device permissions is cut off
![Screen Shot 2022-03-14 at 8 22 47 AM](https://user-images.githubusercontent.com/13563002/158204268-e952fe82-b004-477b-b23a-e1078a76a395.png)

## Errors and logs
None

## Expected behavior
The full focus ring around the button should be visible

## User-facing consequences
It does not match the other instances of `KButton`

## Steps to reproduce
1. Sign in as admin
2. In "Device" click on the "Permissions" tab
3. Use keyboard navigation throughout the "Permissions" tab

## Context
Kolibri 0.15
MacOS
Chrome

**Comment by Abhishek-Punhani** [comment]
Hi @sairina,

I was reviewing this issue, but I am able to see the full focus ring while navigating the page with the keyboard on both Chrome and Firefox. Could you provide more details or guide me on reproducing the issue?

Thanks!

https://github.com/user-attachments/assets/8e51b2a8-0526-4449-a54f-831376c7620a

**Comment by MisRob** [comment]
Hi @Abhishek-Punhani, thank you for checking on it. This is quite an old issue so it's possible it's not the case anymore. Would you be able to test it on different screen sizes as well? If it looks fine, we could close.

**Comment by Abhishek-Punhani** [comment]
Hi @MisRob , I tried to inspect all screen sizes, and I don't see any cut-off focus ring around KButton
https://github.com/user-attachments/assets/88c165be-9a8a-4782-9133-d2b47331cc21

**Comment by MisRob** [comment]
Thank you @Abhishek-Punhani!  Good news. I confirmed this on my local development server too so we can close now.
